### PR TITLE
Cachebust

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: server/lib/fetch-capi-json.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 
+const url = require('url');
+
 module.exports = (endpoint) => {
+
+	// Add cachebust to all CAPI requests, as intermediate caches return stale content despite an update notification
+	const parsed = url.parse(endpoint, true);
+	delete parsed.search;
+	parsed.query.cb = Date.now();
+	endpoint = url.format(parsed);
```
Comment (matthew-andrews) [inline server/lib/fetch-capi-json.js]: thank you for all your effort here, but would you mind restricting this just to the CAPI v1 requests?

here:-
https://github.com/Financial-Times/next-es-interface/blob/master/server/models/article-v3.js#L18

Comment (georgecrawford) [inline server/lib/fetch-capi-json.js]: I had a feeling you might say that. Better now?

File: tests/server/models/article-v3-spec.js [hunk 2]
```diff
@@ -22,6 +22,31 @@ const ARTICLE_ID = 'cf750772-6c4d-11e5-8171-ba1968cf791a';
...
+			})
+			.reply(200, fixtureV1);
+
+		const requestV2 = nock(CAPI_V2_URL)
+			.get(`/${ARTICLE_ID}`)
+			.times(1)
+			.reply(200, fixtureV2);
+
+		return instance(ARTICLE_ID)
+			.then(response => {
```
Comment (matthew-andrews) [inline tests/server/models/article-v3-spec.js]: response is defined by not used :] (linting)

**Comment by matthew-andrews** [comment]
i think circle has helpfully found a linting error but otherwise lovey +1

**Comment by georgecrawford** [comment]
Yeah - I've only now discovered `make install` to add the `.eslintrc.json` file.

**Comment by matthew-andrews** [comment]
quick quick merge merge

**Pull request merged**